pkg/cni/cilium/cilium.go

```diff
@@ -114,8 +114,21 @@ func ApplicationDefinitionReconciler(config *kubermaticv1.KubermaticConfiguratio
 						},
 					},
 				},
+			},
```

Comment:
This closes `app.Spec.Versions` already. You'll need to remove it and then possibly close it after the 1.13.7 entry.